src/diffusers/pipelines/deepfloyd_if/pipeline_if_img2img.py

```diff
@@ -139,6 +138,7 @@ class IFImg2ImgPipeline(DiffusionPipeline, LoraLoaderMixin):
 
     _optional_components = ["tokenizer", "text_encoder", "safety_checker", "feature_extractor", "watermarker"]
     model_cpu_offload_seq = "text_encoder->unet"
+    _exclude_from_cpu_offload = ["watermarker"]
```

Comment:
`whatermarker` does not work with the sequential_cpu_offload hooks, since they are very small, I'm just excluding it